src/utils.js

```diff
@@ -403,3 +404,12 @@ export const isPushNotificationAvailableForUser = (state) => (
 export const getNanoContractFeatureToggle = (state) => (
   state.featureToggles[NANO_CONTRACT_FEATURE_TOGGLE]
 );
+
+/**
+ * Get timestamp in specific format.
+ *
+ * @param {number} timestamp
+ *
+ * @returns {string} formatted timestamp
+ */
+export const getTimestampFormat = (timestamp) => moment.unix(timestamp).format(t`DD MMM YYYY [•] HH:mm`)
```

Comment:
This is copying what we have in the `src/models.js` file to the utils. You should at least change the models file to use the utils as well.